debian/control

```diff
@@ -10,6 +10,7 @@ Build-Depends: autoconf,
                automake,
                autotools-dev,
 	       libbz2-dev,
+               liblz4-dev,
```

Comment:
this must also go in the ceph.spec.in BuildDepends so that RPM based platforms install it (centos being one of them ;-)